src/app/clients/actionBar/ActionsModal.tsx

```diff
@@ -0,0 +1,123 @@
+"use client";
+
+import React, { useEffect, useState } from "react";
+import styled from "styled-components";
+import Modal from "@/components/Modal/Modal";
+import dayjs, { Dayjs } from "dayjs";
+import { ClientsTableRow } from "@/app/clients/getClientsTableData";
+import { Button } from "@mui/material";
+import FreeFormTextInput from "@/components/DataInput/FreeFormTextInput";
+import { DatePicker } from "@mui/x-date-pickers";
+
+interface ActionsModalProps {
+    isOpen: boolean;
```

Comment:
For props that are only for the Modal, you can extend `React.ComponentProps<typeof Modal>` in your interface instead